jenkins/deploy-clio-server.sh

```diff
@@ -156,8 +156,11 @@ render_ctmpls() {
 }
 
 activate_service_account() {
+  ENV=${1}
+
+  echo "Activating Service Account"
```

Comment:
quotes have no effect here (but people seem to like them)
```suggestion
  echo Activating Service Account
```